src/lib/services/settings-service.ts

```diff
@@ -25,6 +25,7 @@ export const fetchSettings = async (request = fetch): Promise<Settings> => {
     codec: {
       endpoint: settingsResponse?.Codec?.Endpoint,
       passAccessToken: settingsResponse?.Codec?.PassAccessToken,
+      passCredentials: settingsResponse?.Codec?.PassCredentials,
```

Comment:
Are we adding this to the settings api through an env var?

Reply:
Yeah we'll want to do that like we do for the access token.